.github/workflows/test.yml

```diff
@@ -24,28 +27,25 @@ jobs:
         uses: gradle/gradle-build-action@v2.6.0
 
       - name: Execute Gradle build
-        run: ./gradlew clean platform:test
+        run: ./gradlew clean test
 
-      - name: Report
-        uses: dorny/test-reporter@v1
-        if: always()
-        with:
-          name: Test results
-          path: ${{ github.workspace }}/platform/build/test-results/test/TEST-*.xml
-          reporter: java-junit
-          only-summary: true
-
-      - name: Report test coverage
```

Comment:
Do we not want test coverage?

Reply:
We have that with a separate reporter, conforming with DI

Reply:
Reverting this